README.md

```diff
@@ -321,7 +321,7 @@ To update Testify to the latest version, use `go get -u github.com/stretchr/test
 Supported go versions
 ==================
 
-We support the three major Go versions, which are 1.11, 1.12, and 1.13 at the moment.
+We support the Go versions 1.13 and greater.
```

Comment:
Thanks for doing this. Do you mind explicitly listing them? (i.e. 1.13 and 1.14).

Reply:
Not at all. Changed the wording to match the original, with the exception of only listing two major versions.